README.md

```diff
@@ -1,99 +1,254 @@
+> 📚 More detailed documentation can be found in the [doc](src/improvements/doc/) directory.
 
-Tasks include:
+## Repository Structure
 
-- new contract deployments
-- contract upgrades
-- onchain configuration changes
+The repository is organized as follows:
 
-Effectively any significant change to the state of the network, requiring authorization to execute, should be considered a task.
+```
+superchain-ops/
+└── src/
+    └── improvements/ 
+       ├── template/     # Solidity template contracts (Template developers create templates here)
+       └── doc/          # Detailed documentation
+       └── tasks/        # Network-specific tasks
+            ├── eth/     # Ethereum mainnet tasks (Task developers create tasks here)
+            └── sep/     # Sepolia testnet tasks  (Task developers create tasks here)
 
-## Directory structure
+```
 
-Top level directory names should be the [EIP-3770](https://eips.ethereum.org/EIPS/eip-3770) short name for the network (see [shortNameMapping.json](https://chainid.network/shortNameMapping.json))
+## Quick Start
 
-Each task should contain the following:
+> ⚠️ **IMPORTANT**: **Do not** update `mise` to a newer version unless you're told to do so by the maintainers of this repository. We pin to specific allowed versions of `mise` to reduce the likelihood of installing a vulnerable version of `mise`. You **must** use the `install-mise.sh` script to install `mise`.
 
-- `README.md`: A brief markdown file describing the task to be executed.
-- `Validation.md`: A markdown file describing and justifying the expected state changes for manual validation by multisig signers.
-- A foundry script that implements post upgrade assertions, depending on the nature of the ceremony it should be called either:
-  - `SignFromJson.s.sol`: If the ceremony is for a Safe owned by EOA signers.
-  - `NestedSignFromJson.s.sol`: If the ceremony is for a Safe owned by other Safe's.
-- `input.json`: A JSON file which defines the specific transaction for the task to be executed. This file may either be generated automatically or manually created.
-- `.env`: a place to store environment variables specific to this task.
+1. Install dependencies:
+```bash
+cd src/improvements/
+./script/install-mise.sh # Follow the instructions in the log output from this command to activate mise in your shell.
+mise trust ../../mise.toml
+mise install
+just --justfile ../../justfile install
+```
 
-## Installation
+> For more information on `mise`, please refer to the [CONTRIBUTING.md](./CONTRIBUTING.md) guide.
 
-The following instructions are for MacOS, but should be similar for other operating systems.
+2. Run tests:
+```bash
+cd src/improvements/
+forge test # Run solidity tests.
+```
 
-For each of these steps, if you already have some version of the software installed, it should be safe to skip it.
+3. Create a new task:
+```bash
+cd src/improvements/
```

Comment:
I don't know if this is intended by in the step 2 we are already making a `cd src/improvements/` so either we need to go back to `cd -` either this is not necessary here.

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/1163/commits/3127ada3227872bf6143bb824ada64dc77cfc31a